llmfoundry/utils/checkpoint_conversion_helpers.py

```diff
@@ -183,11 +185,11 @@ def _convert_weight_to_ft_each(save_dir: str, infer_gpu_num: int,
             split_vals[j].tofile(save_path)
             if config['no_bias']:
                 _write_zero_bias(tensor_name, save_path,
-                                 split_vals[j].shape[-1])
+                                 split_vals[j].shape[-1], np_weight_data_type)
 
     elif tensor_name.find('mlp.dense_h_to_4h.bias') != -1:
         assert data.shape == (
-            config['mlp_ratio'] *
+            config['expansion_ratio'] *
```

Comment:
Will this fail for older checkpoints? e.g., current mpt7b/mpt40b hf checkpoints?

Reply:
we pass hf_config (HF config dict) to this function; for mpt models, we use expansion_ratio. I'm surprised this was working before.